src/components/moj-messages/template.njk

```diff
@@ -0,0 +1,30 @@
+<div id="{{ params.id | default('messages') }}" class="moj-messages-container {{- ' ' + params.classes if params.classes}}" {%- if (params.label) %} aria-label="{{ params.label }}"{% endif %} {%- for attribute, value in params.attributes -%} {{ attribute }}="{{ value }}"{% endfor %}>
+
+  <div class="moj-message-list">
+
+    {%- for item in params.items %}
+
+      {%- if previous_timestamp != item.timestamp | mojDate('date') %}
+
+        <time class="moj-message-list__date" datetime="{{ item.timestamp | mojDate('date') }}">
+          {{- item.timestamp | mojDate('date') -}}
+        </time>
+
+      {% endif -%}
+
+      <div {%- if (item.id) %} id="message_{{ item.id }}"{% endif %} class="moj-message-item moj-message-item--{{ item.type }}">
```

Comment:
How will the user know a new message has been added in real time? Is this a real time thing? Have you considered aria live regions?

Reply:
In its current incarnation, this isn't a realtime component. Having said that, there is no reason why it could be enhanced.